src/vault/use-cases/keys/create_key.go

```diff
@@ -69,8 +69,8 @@ func (uc *createKeyUseCase) Execute(ctx context.Context, namespace, id, algo, cu
 			return nil, errors.InvalidParameterError(errMessage)
 		}
 
-		key.PrivateKey = hex.EncodeToString(crypto2.FromECDSA(privKey))
-		key.PublicKey = hexutil.Encode(crypto2.FromECDSAPub(&privKey.PublicKey))
+		key.PrivateKey = base64.StdEncoding.EncodeToString(crypto.FromECDSA(privKey))
```

Comment:
I think it would be good to encapsulate encode and decode operation under a util so we can easily move from one type to another without modifying it all over our codebase, WDYT?

Reply:
done